settings/src/utilities.js

```diff
@@ -17,11 +17,17 @@ export function useUser( userId ) {
 	const hasPrimaryProvider = !! availableProviders.filter( ( provider ) =>
```

Comment:
This loop with inner loop is potentially expensive, and we already have the answer from `totpEnabled` and `webAuthnEnabled`. Suggest replacing with `const hasPrimaryProvider = totpEnabled || webAuthnEnabled;`.

In theory the tests should still pass.

Reply:
Commit added: [90306c5](https://github.com/WordPress/wporg-two-factor/pull/197/commits/90306c546408bb481b2ea40e1902d5d11fb2a5d3)